android/library/src/main/java/io/logto/android/authflow/webview/WebViewAuthFlow.kt

```diff
@@ -23,7 +23,7 @@ class WebViewAuthFlow(
 ) {
 
     private val codeVerifier: String = PkceUtil.generateCodeVerifier()
-    private val logtoClient = LogtoClientBuilder(logtoConfig).build()
+    private val logtoService = LogtoService.create(logtoConfig.oidcEndpoint)
```

Comment:
is it the Kotlin convention? or we can just use [constructors](https://kotlinlang.org/docs/classes.html#creating-instances-of-classes) to create a instance

Reply:
It depends on retrofit. `LogtoService` is `interface`, we should create instances by retrofit.
This pattern following Google's paractice about androidx.
Ref: [sunflower](https://github.com/android/sunflower/blob/main/app/src/main/java/com/google/samples/apps/sunflower/api/UnsplashService.kt)

Reply:
sry didn't see `interface`